templates/documentation/indicators.html

```diff
@@ -14,7 +14,67 @@ <h2>Climate Indicators</h2>
 
 <h3>Service endpoints</h3>
 
-<h4>All climate indicator variables</h4>
+<h4>CMIP6 climate indicator variables</h4>
+
+<p>
+  Query CMIP6 climate indicator variables for all models and scenarios.
+</p>
+
+<table class="endpoints">
+  <thead>
+    <tr>
+      <th class="endpoint-label">Endpoint</th>
+      <th class="endpoint-url">Example URL</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Annual mean values for all climate indicator variables at a given latitude and longitude.</td>
```

Comment:
```suggestion
      <td>Yearly climate indicator values at a given latitude and longitude.</td>
```
Just a bit more precise, as these are not annual means